packages/flutter/test/material/expand_icon_test.dart

```diff
@@ -312,4 +312,23 @@ void main() {
     iconTheme = tester.firstWidget(find.byType(IconTheme).last);
     expect(iconTheme.data.color, equals(Colors.cyan));
   });
+
+  testWidgets('Expand Icon test', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+      home: Scaffold(
+          body: ExpandIcon(
+        onPressed: (v) {},
+        iconData: Icons.hearing,
+      )),
+    ));
```

Comment:
Formatting looks off here.

```suggestion
    await tester.pumpWidget(MaterialApp(
      home: Scaffold(
        body: ExpandIcon(
          onPressed: (v) {},
          iconData: Icons.hearing,
        ),
      ),
    ));
```